### ISSUE TITLE
OSX - Prevent MPV from opening off the screen?

### ISSUE THREAD

**Issue by victorhooi** [mpv-player/mpv#1912]
I am using `mpv` 0.9.0 on OSX 10.10.4 (compiled via `homebrew`).

Often, when I open a video file, if the video resolution is large, part of the video will render off screen - I then need to drag the `mpv` window back onto screen, then click the green OSX maximise button to get it to shrink back to filling the screen.

Is it possible to add a configuration flag that will "cap" `mpv` at the available screen resolution? That is, if a video is higher resolution than the monitor resolution, it will just default to maximised? That would seem like more intuitive behaviour than having the video appear half off the screen.

On a similar note, would it also be possible to add a configuration flag to always default to maximised? (That is, if the video is *smaller* than the available resolution, then we will maximise it etc.).

**Comment by selsta** [comment]
`autofit-larger=90%x90%`

**Comment by victorhooi** [comment]
@selsta Awesome, thanks - that's perfect =). I've just tested it, and it does stop the video window from appearing off screen.

Just out of curiosity - why 90%x90%?

(I did try with both 90%x90% and 100%x100% - the 100% still worked, and the window appeared under the OSX dock, so that was fine - it didn't obscure the dock - although in this case, it didn't quite fill things on the vertical, since it had already filled the horizontal axis and I believe it tries to preserve aspect ratio).
